Gemfile

```diff
@@ -5,6 +5,9 @@ source 'https://rubygems.org'
 gemspec
 
 group :test do
-  gem 'rspec',     '~> 3.1.0'
-  gem 'simplecov', '~> 0.9.1'
+  gem 'rspec', '~> 3.1.0'
+
+  if RUBY_ENGINE.eql?('ruby') && RUBY_VERSION >= '2.7'
```

Comment:
@dkubb I recommend to simply drop all EOL rubies from the CI this helps a lot should you incorporate OSX builds which are much slower.

Reply:
@mbj the `RUBY_ENGINE` part is still needed to allow this to work with jruby and truffle ruby (I think, but will verify). It should allow the `RUBY_VERSION` to be removed if I do not target EOL.

I think for now I'm going to keep support for older rubies, but I may revisit this in the future in a separate PR. I was mostly concerned with maintaining parity with the previous release and getting things to pass with modern rubies.